clifford/__init__.py

```diff
@@ -258,17 +260,73 @@ def general_exp(x, max_order=15):
     for i in range(1, max_order):
         if np.any(np.abs(tmp.value) > _settings._eps):
             tmp = tmp*scaled * (1.0 / i)
-            result += tmp
+            result = result + tmp
```

Comment:
What's the motivation for this change?

Reply:
Motivation was that without this numba will not JIT our general_exp function:
```
No implementation of function Function(<built-in function iadd>) found for signature:
iadd(MultiVector(LayoutType(Layout([1, 1, 1], ids=BasisVectorIds.ordered_integers(3), order=BasisBladeOrder.shortlex(3), names=['', 'e1', 'e2', 'e3', 'e12', 'e13', 'e23', 'e123'])), float64), MultiVector(LayoutType(Layout([1, 1, 1], ids=BasisVectorIds.ordered_integers(3), order=BasisBladeOrder.shortlex(3), names=['', 'e1', 'e2', 'e3', 'e12', 'e13', 'e23', 'e123'])), float64))
```